### PR TITLE
update_llpc_test_checks.py: use ';' for comments in .spvasm files

### PR DESCRIPTION
**Pull request by nhaehnle**

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit 22bb3dea211d0a764d5cd0d321a93ba3a8e30479
<details><summary>CTS tests (Failed: 1/207274)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi2x</summary><ul>

<li>Passed: 39103/70473 (55.5%)</li>
<li><details><summary>Failed: 1/70473 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.rasterization.conservative.overestimate.samples_4.triangles.degenerate.max
Stack trace: Flake
```
</p></details></li>

<li>Not Supported: 31369/70473 (44.5%)</li><li>Warnings: 0/70473 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>Passed: 35949/70481 (51.0%)</li>
<li>Failed: 0/70481 (0.0%)</li>
<li>Not Supported: 34532/70481 (49.0%)</li><li>Warnings: 0/70481 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx9</summary><ul>

<li>Passed: 33668/66320 (50.8%)</li>
<li>Failed: 0/66320 (0.0%)</li>
<li>Not Supported: 32652/66320 (49.2%)</li><li>Warnings: 0/66320 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by Flakebi** [comment]
Searching through all file-endings with `find | rev | cut -s -d. -f1 | rev | sort | uniq`, there are also
- `.ll` (`error_reporting/*.ll`), not sure if that’s important for auto-generation
- `.multi-input`, probably we can auto-generate those?
- `.spvas` (only two files that should probably be renamed to .spvasm)

**Comment by nhaehnle** [comment]
Thanks, I added `.ll` as well. I didn't find the `.spvas` files but if they exist, they should be renamed. The `.multi-input` seems like too much of a special-purpose case to me, those tests don't seems like candidates for auto-generated CHECK lines to me.

File: tool/update_llpc_test_checks.py [hunk 1]
```diff
@@ -137,9 +137,17 @@ def add_ir_checks(output_lines, comment_marker, prefix_list, func_dict,
...
+  ('.ll', ';'),
+  # Everything else defaults to '//'
+]
+
 def get_comment_prefix(test_name: str, input_lines):
-  if test_name.endswith('.pipe'):
-    return ';'
+  for suffix, comment_prefix in COMMENT_PREFIXES_BY_FILE_SUFFIX:
+    if test_name.endswith(suffix):
+      return comment_prefix
```
Comment (Flakebi) [inline tool/update_llpc_test_checks.py:150]: Thanks, maybe make it a dictionary?
```python
COMMENT_PREFIXES_BY_FILE_EXT = {
  '.pipe': ';',
  '.spvasm': ';',
  '.ll': ';',
  # Everything else defaults to '//'
}

def get_comment_prefix(test_name: str, input_lines):
  ext = os.path.splitext(test_name)[1]
  return COMMENT_PREFIXES_BY_FILE_EXT.get(ext, '//')
```

Comment (nhaehnle) [reply tool/update_llpc_test_checks.py]: Nice one :)

**Comment by amdvlk-admin** [comment]
### Test summary for commit e920189b84ce7fb272f54e461862b97789f7f25e
<details><summary>CTS tests (Failed: 2/207247)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi2x</summary><ul>

<li>Passed: 39097/70464 (55.5%)</li>
<li><details><summary>Failed: 2/70464 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.rasterization.conservative.overestimate.samples_8.triangles.degenerate.max
Stack trace: Fail

FAILURE: dEQP-VK.rasterization.conservative.overestimate.samples_8.triangles.degenerate.min
Stack trace: Fail
```
</p></details></li>

<li>Not Supported: 31365/70464 (44.5%)</li><li>Warnings: 0/70464 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>Passed: 35944/70472 (51.0%)</li>
<li>Failed: 0/70472 (0.0%)</li>
<li>Not Supported: 34528/70472 (49.0%)</li><li>Warnings: 0/70472 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx9</summary><ul>

<li>Passed: 33663/66311 (50.8%)</li>
<li>Failed: 0/66311 (0.0%)</li>
<li>Not Supported: 32648/66311 (49.2%)</li><li>Warnings: 0/66311 (0.0%)</li>
</ul></details>


</ul></details>

**Review by Flakebi — APPROVED** [review]

**Review by amdrexu — APPROVED** [review]

**Comment by nhaehnle** [comment]
retest this please

**Pull request merged**